integration/util.py

```diff
@@ -140,15 +141,15 @@ def processEnded(self, reason):
 
     def outReceived(self, data):
         data = str(data, sys.stdout.encoding)
-        sys.stdout.write(data)
+        sys.stdout.write(self.name + data)
```

Comment:
Does this need to do line-splitting? e.g. to ensure that if `data` has multiple or partial lines the output still looks good?

Reply:
I looked at the output and it seemed fine as is.